src/Security/RememberLoginHash.php

```diff
@@ -80,7 +80,21 @@ class RememberLoginHash extends DataObject
     private static $force_single_token = false;
 
     /**
-     * The token used for the hash
+     * If true, the token will be replaced during session renewal. This can cause unexpected
+     * logouts if the new token does not reach the client (e.g. due to a network error).
+     *
+     * This can be disabled as of CMS 5.3, and renewal will be removed entirely in CMS 6.
+     *
+     * @config
+     *
+     * @var bool
+     */
+    private static $replace_token_during_session_renewal = true;
```

Comment:
```suggestion
     * @deprecated 5.3.0 Will be removed without equivalent functionality
     */
    private static bool $replace_token_during_session_renewal = true;
```
We don't use `@config` anymore (all private statics are considered config unless otherwise stated).

We prefer strict typehints over PHPDoc, even for config. The config system doesn't enforce types right now but that's on my wishlist for a future major release.

We also need to deprecate the config property so we remember to remove it for CMS 6.